app/views/courses/index.html.erb

```diff
@@ -1,21 +1,23 @@
-<div class="container">
-<% if @search_name %>
-  <div class="result-title">
-    <h1>List of all the classes with "<%= @search_name %>"</h1>
-  </div>
-<% elsif @category_name %>
-  <div class="result-title">
-    <h1>List of all the classes in : <b><%= @category_name %></b></h1>
-  </div>
-<% else %>
-  <div class="result-title">
-    <h1>List of all the classes</h1>
-  </div>
-<% end %>
+<div class="<%= @markers.any? ? 'container-map-card' : 'container' %>">
+
+  <div class="<%= @markers.any? ? 'courses-list' : '' %>">
+    <% if @search_name %>
+      <div class="result-title">
+        <h1>List of all the classes with "<%= @search_name %>"</h1>
+      </div>
+    <% elsif @category_name %>
+      <div class="result-title">
+        <h1>List of all the classes in : <b><%= @category_name %></b></h1>
+      </div>
+    <% else %>
+      <div class="result-title">
+        <h1>List of all the classes</h1>
+      </div>
+    <% end %>
 
   <% @courses.each do |course| %>
     <%= link_to course_path(course), {class: "without-text-decoration"} do%>
-      <div class="card-results-courses">
+      <div class="<%= course.online? ? 'card-results-courses-online' : 'card-results-courses' %>">
```

Comment:
Nao entendo bem essa linha, faz o que ? O codigo ta dentro da class ?

Reply:
A gente aprendeu essa tática com o Matheus ontem. 
Basicamente ele teste uma condicional e então determina qual classe será.
Então teremos uma classe de configuração se o curso for online, e outra se o curso não for online.